### PR TITLE
#191 Fix location in slot

### PR DESCRIPTION
**Pull request by EugeneValko**

### CODE REVIEW

**Review by va-dym-ko-val — APPROVED** [review]

File: frontend/src/app/modules/availability/slot/slot.component.html [hunk 1]
```diff
@@ -88,9 +88,9 @@
             </div>
         </div>
         <div class="meeting-details">
-            <p *ngIf="hasOwner">{{ slot.size }} min, {{ slot.authorName }}, {{ LocationType[slot.locationType] }}</p>
+            <p *ngIf="hasOwner">{{ slot.size }} min, {{ slot.authorName }}, {{ locationTypeValues[+slot.locationType] }}</p>
```
Comment (tatianahutii) [inline frontend/src/app/modules/availability/slot/slot.component.html]: This logic looks strange

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/core/helpers/country-label-mapping.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-    [Country.Ukraine]: 'Ukraine',
-    [Country.Poland]: 'Poland',
-    [Country.Sweden]: 'Sweden',
-    [Country.Italy]: 'Italy',
-    [Country.Uganda]: 'Uganda',
-};
+export function CountryLabelMapping(country: Country | string): string {
+    switch (country) {
+        case Country.UnitedStates:
+        case 'UnitedStates':
```
Comment (tatianahutii) [inline frontend/src/app/core/helpers/country-label-mapping.ts]: Why there two cases?

File: frontend/src/app/core/helpers/location-type-mapping.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { LocationType } from '@shared/enums/locationType';
+
+export function LocationTypeMapping(locationType: LocationType | string): string {
+    switch (locationType) {
+        case LocationType.GoogleMeet:
+        case 'GoogleMeet':
```
Comment (tatianahutii) [inline frontend/src/app/core/helpers/location-type-mapping.ts]: see the previous comment

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/core/helpers/country-label-mapping.ts (hunk 2 above)
Comment (EugeneValko) [reply frontend/src/app/core/helpers/country-label-mapping.ts]: Fixed

File: frontend/src/app/core/helpers/location-type-mapping.ts (hunk 3 above)
Comment (EugeneValko) [reply frontend/src/app/core/helpers/location-type-mapping.ts]: Fixed

**Review by tatianahutii — APPROVED** [review]
LGTM

File: backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs [hunk 4]
```diff
@@ -23,7 +24,12 @@ public static void RegisterCustomServices(this IServiceCollection services, ICon
             services
                 .AddControllers()
                 .AddNewtonsoftJson(options => options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
-
+            services
+                .AddControllers()
+                .AddNewtonsoftJson(options =>
+                {
+                    options.SerializerSettings.Converters.Add(new StringEnumConverter());
+                });
```
Comment (apofis96) [inline backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs]: Combine to one call

**Review by apofis96 — CHANGES_REQUESTED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**